static-site/src/components/ListResources/index.tsx

```diff
@@ -310,3 +310,58 @@ const useShowcaseCards = (cards?: FilterCard[], groups?: Group[]) => {
   }, [cards, groups]);
   return restrictedCards;
 }
+
+const cardWidthHeight = 160; // pixels
+
+const CardOuter = styled.div`
```

Comment:
_non-blocking_

I'm not 100% convinced by this change since I'd be worried about the splash and ListResources card styling diverging. Would it make sense to pull the card styling out as a separate shared component? (Maybe confusing with the [existing Cards component](https://github.com/nextstrain/nextstrain.org/tree/master/static-site/src/components/Cards)...)

Reply:
Oh actually, there's a lot of overlap in the styling with the [existing Cards styles](https://github.com/nextstrain/nextstrain.org/blob/05aba2bc93ef855de7b0c932b977d98c11b40408/static-site/src/components/Cards/styles.tsx), would it make sense to just have a single centralized Card component?

Reply:
I think it's fine to have separate copies, at least while we figure out what design works best for each page. In my experience so far, the idea of a centralized Card design has been limiting and it's easiest to have different components (hence the parameterization of `CardComponent` in 1414985c92488423035a1470534c22a8ef479595 and follow-up to that in this PR as 9beac0fa2c324b326587063e0f2e3ef48d9ac653).

> I'd be worried about the splash and ListResources card styling diverging

I've already made that divergence happen in #922. I think we've all agreed that the new splash page card design looks good on that page. This design doesn't make sense for the ListResources showcase pages (`/pathogens`, `/staging`). While some parts of the design may work well, there's still some obvious differences (e.g. no need for the "maintainer" icon since all those are maintained by the Nextstrain team).

I haven't touched the existing Cards component (`static-site/src/components/Cards`) but after these PRs it's probably time to re-scope that to its only usage which is on the `/groups` page. It also doesn't make sense to apply the new splash page card design (linking to datasets/narratives) to the groups cards (linking to group pages).

Reply:
Ah, I haven't gotten to that PR yet, but this separation makes sense in that context 👍

Reply:
> I haven't touched the existing Cards component (static-site/src/components/Cards) but after these PRs it's probably time to re-scope that to its only usage which is on the /groups page. It also doesn't make sense to apply the new splash page card design (linking to datasets/narratives) to the groups cards (linking to group pages).

The cards UI currently in use on `/groups` (the one using blank colours rather than images) isn't something we need to preserve. That is to say, we can re-organise the source files to make it clear it's only used in one place, but we shouldn't invest time developing it without considering alternatives.

Reply:
> we can re-organise the source files to make it clear it's only used in one place

Added as a task in https://github.com/nextstrain/nextstrain.org/issues/934